src/main/java/dev/codesupport/web/common/util/StringUtils.java

```diff
@@ -0,0 +1,36 @@
+package dev.codesupport.web.common.util;
+
+import lombok.NonNull;
+
+import java.util.StringJoiner;
+
+/**
+ * Utilities for common String manipulation logic
+ */
+public class StringUtils {
+
+    private StringUtils() {
+
+    }
```

Comment:
Empty?

Reply:
Utility classes have private constructors to keep them from being initialized, since they have no instance methods.  For this reason the constructor is also empty, since there are no instance based properties either.